app/client/src/components/shared/CharacteristicsSelect.tsx

```diff
@@ -1,41 +1,57 @@
-import uniqueId from 'lodash/uniqueId';
-import { useContext, useMemo, useState } from 'react';
+import { useContext, useMemo } from 'react';
 import Select from 'react-select';
 import { css } from 'styled-components/macro';
 // components
+import { GlossaryTerm } from 'components/shared/GlossaryPanel';
 import MenuList from 'components/shared/MenuList';
 // contexts
 import { LocationSearchContext } from 'contexts/locationSearch';
 // utils
 import { useMonitoringLocations } from 'utils/hooks';
+// styles
+import { groupHeadingStyles, reactSelectStyles } from 'styles/index';
 
 export default CharacteristicsSelect;
 export function CharacteristicsSelect({
-  label,
   selected,
   onChange,
 }: CharacteristicsSelectProps) {
   const { monitoringPeriodOfRecordStatus } = useContext(LocationSearchContext);
   const { monitoringLocations } = useMonitoringLocations();
 
```

Comment:
Related to below, I think the explicit label should be preferred.
```suggestion
  const [inputId] = useState(uniqueId('characteristic-select-'));
```